base/Dockerfile

```diff
@@ -21,28 +21,64 @@ LABEL License="MIT License"
 # Start s6
 ENTRYPOINT [ "/init" ]
 
+ENV \
```

Comment:
These would be better exposed as build arguments rather than environment variables, as setting them at runtime has no effect. Additionally, using `ENV` adds a layer to the image.

Reply:
Good call. Fixed with https://github.com/Islandora-Devops/isle-buildkit/pull/380/commits/1b1799abaee6f583e4715dec286decd8ef141845